openlibrary/plugins/openlibrary/lists.py

```diff
@@ -323,31 +324,23 @@ class list_seed_yaml(list_seeds):
     encoding = "yml"
     content_type = 'text/yaml; charset="utf-8"'
 
-
-class list_editions(delegate.page):
```

Comment:
`list_editions` (not the json endpoint) appears to be dead code as there is no such template `type/list/editions.html`

`r"(/people/\w+/lists/OL\d+L)/editions"` i.e.
https://openlibrary.org/people/mekBot/lists/OL104041L/Popular_Books/editions?debug=true